sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/_connection.py

```diff
@@ -792,8 +798,21 @@ def listen(self, wait=False, batch=1, **kwargs):
         except Exception:  # pylint:disable=try-except-raise
             raise
 
-    def create_session(self, **kwargs):
-        # type: (Any) -> Session
+    def create_session(
+            self,
+            *,
+            name: Optional[str] = None,
+            next_outgoing_id: int = 0,
+            incoming_window: int = 1,
+            outgoing_window: int = 1,
+            handle_max: int = 4294967295,
+            offered_capabilities: Optional[List[str]] = None,
+            desired_capabilities: Optional[List[str]] = None,
+            properties: Optional[Dict[str, Any]] = None,
```

Comment:
I don't think these are actually `Any` - I think there might be a supported type set here.

Reply:
for properties, they are [defined](https://docs.oasis-open.org/amqp/core/v1.0/os/amqp-core-transport-v1.0-os.html#type-fields) as a `map of fields` where the keys are `symbols` ( strings ) but the values are not restrained